install/kubernetes/tetragon/templates/operator_servicemonitor.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     {{- with .Values.tetragonOperator.prometheus.serviceMonitor.labelsOverride}}
     {{- toYaml . | nindent 4 }}
     {{- else }}
-    {{- include "tetragon.labels" . | nindent 4 }}
+    {{- include "tetragon-operator.labels" . | nindent 4 }}
     {{- end }}
   name: {{ .Release.Name }}
```

Comment:
Should we also change the name? (e.g.: `name: {{ .Release.Name }}-operator`)

Reply:
good catch, yes